src/status_im/utils/types.cljs

```diff
@@ -1,13 +1,15 @@
 (ns status-im.utils.types
-  (:require [cognitect.transit :as transit]))
+  (:require
+   [cljs-bean.core :as clj-bean]
+   [cognitect.transit :as transit]))
 
 (defn to-string [s]
   (if (keyword? s)
     (name s)
     s))
 
 (defn clj->json [data]
```

Comment:
Do we need to rename this function or it will work with clojure data structures as well?

Reply:
I think it just works, but will double check